Sources/TokamakCore/Modifiers/ViewModifier.swift

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+public protocol ViewModifier {
+    typealias Content = _ViewModifier_Content<Self>
+    associatedtype Body : View
+    func body(content: Content) -> Self.Body
+}
+
+public struct _ViewModifier_Content<Modifier> : View where Modifier : ViewModifier {
```

Comment:
I wonder if this could be replaced with the [`ModifiedContent`](https://github.com/twostraws/swiftui-changelog/blob/9b31fe343f5cab36d8dae42fb5ac9c301d7b0945/generated-interface.swift#L8899) struct to match the SwiftUI API more closely?

Reply:
SwiftUI actually has `_ViewModifier_Content` and `ModifiedContent`. They use `_ViewModifier_Content` for the type alias `Content` in the `ViewModifier`. `ModifiedContent` is a bit different, as it applies to more than views (such as other `ViewModifiers`). I can implement `ModifiedContent` separately, or switch it to use `ModifiedContent`...

Reply:
Oh, I didn't know that, interesting that `_ViewModifier_Content` is ommitted from the auto-generated interfaces. Totally fine if `ModifiedContent` is added separately then.

Reply:
I'm fairly sure the interface will automatically hide the underscored names.

Reply:
Here's how I found it: https://stackoverflow.com/questions/56528833/is-there-a-way-to-find-actual-swiftui-api-documentation-and-not-just-the-develo/56530199#56530199
Inside the Xcode app: `Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk/System/Library/Frameworks/SwiftUI.framework/Modules/SwiftUI.swiftmodule/arm64.swiftinterface`